src/glue.js

```diff
@@ -294,8 +301,8 @@ function updateI18NClasses () {
 	var base = 'enyo-locale-';
 
     // Remove old style definitions (hack style becouse enyo.dom doesn't have methods like enyo.dom.getBodyClasses, enyo.dom.removeBodyClass)
-    if (document && document.body && document.body.className && document.body.className) {
```

Comment:
className is truthy. OH WAIT! Was className truthy?!?!?! Damn my short term memory.